### ISSUE TITLE
Error in "STEP 2 OF 2. SET UP GAME PATH"

### ISSUE THREAD

**Issue by Antipov111** [ReZeroE/StarRail#29]
![изображение](https://github.com/user-attachments/assets/436642a4-dedc-4e10-9673-dceb3634e2a3)

**Comment by ReZeroE** [comment]
This error is thrown because the program can't locate the game on your computer (it looks for `.../Star Rail/Game/StarRail.exe`).

I just now noticed that the installation directory has changed with the new Hoyo launcher. I'll need to reinstall the game using the new launcher and then update the path the program uses to locate the game.

Thanks for reporting.

**Comment by ReZeroE** [comment]
Issue resolved in version `1.0.2`.

```
pip uninstall starrail
pip install starrail==1.0.2

starrail configure
```

**Comment by Antipov111** [comment]
![изображение](https://github.com/user-attachments/assets/641344c5-0f7c-431c-a4da-0e588f8041e0)
![изображение](https://github.com/user-attachments/assets/b3bdf17b-1ecc-4c6b-a7a1-058a8cfc10f9)
![изображение](https://github.com/user-attachments/assets/f5d964ad-8bb4-44ec-b230-8f39a1e28248)
![изображение](https://github.com/user-attachments/assets/09aa8b6e-9652-4f36-a364-68c88b6f05bc)
![изображение](https://github.com/user-attachments/assets/933c4202-ad9f-44b1-9027-4672c29da2fb)

**Comment by ReZeroE** [comment]
All the issues have been fixed (PR #31).

I couldn't reproduce the `dividing by zero` exception (from image 1). Theoretically, the divisor should never be zero because it corresponds to the game window's height/width. Nevertheless, I added a try-catch block that skips this step if it occurs for now, as it doesn't impact the program's execution.

Thanks again for reporting. 

```
pip uninstall starrail
pip install starrail==1.0.4
```
